faq/README.md

```diff
@@ -57,3 +57,10 @@ higher latencies.
 We therefore urge users to carefully choose upfront the consistency level that works best for their
 use cases. If there is a legitimate reason to downgrade and retry, that should be handled by the
 application code.
+
+### I want to set a date on a bound statement, where did `setTimestamp()` go?
+
+The driver now uses Java 8's improved date and time API. CQL type `timestamp` is mapped to
```

Comment:
Good idea for that change. I've also had that problem when rewriting examples from `3.x` to `4.x`. The `timestamp*` methods from `3.x` are now `instant*`.